simulation_parameters/Constants.cs

```diff
@@ -600,6 +600,16 @@ public static class Constants
     public const int KIBIBYTE = 1024;
     public const int MEBIBYTE = 1024 * KIBIBYTE;
 
+    /// <summary>
+    ///   The duration for which a save is considered recently performed.
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     Not a const because TimeSpan is not a primitive.
+    ///   </para>
+    /// </remarks>
+    public static readonly TimeSpan RecentSaveTime = TimeSpan.FromSeconds(10);
```

Comment:
I'd increase this to 15 for now, and we can tweak this later. Because I think some people might deliberate a bit after saving the game before finding the exit button.